ggcore/sdk_exceptions.py

```diff
@@ -2,9 +2,13 @@ class SdkException(Exception):
     """ Top level custom sdk exception """
 
 
-class SdkInvalidClient(Exception):
+class SdkInvalidClient(SdkException):
     """ The client requested is invalid """
 
 
-class SdkAuthTypeException(Exception):
-    """ """
+class SdkAuthTypeException(SdkException):
+    """ Tried using an invalid auth type """
```

Comment:
Minor nitpick, but I don't think you need to include extra whitespaces in these comment strings after+before triple quotes

Reply:
noted, I'll make this change. Thanks!